packages/core/useWebSocket/index.md

```diff
@@ -87,3 +87,14 @@ const { status, data, send, open, close } = useWebSocket('ws://websocketurl', {
   protocols: ['soap'], // ['soap', 'wamp']
 })
 ```
+
+### Adjusting url
+It is possible to adjust the url, e.g. for parametrizing the websocket:
+```js
+import { useWebSocket } from '@vueuse/core'
+
+const ws = useWebSocket('ws://websocketurl?jwt=123', {
```

Comment:
Very nice! This websocket address becomes completely optional?

Reply:
It's rather the fallback as there needs to be an address for immediate and retries

Reply:
Ah yes, immediate needs to be false indeed!